streaming-binlog/src/main/java/org/openmrs/analytics/FhirConverter.java

```diff
@@ -127,4 +138,50 @@ GeneralConfiguration getEventsToFhirConfig(String fileName) throws IOException {
 			return generalConfiguration;
 		}
 	}
+	
+	private String getUuid(String parentTable, String parentForeignKey, String childPrimaryKeyValue, Map payload) {
```

Comment:
Can you please implement some unit-tests that cover this method. Either test this directly OR add examples that test `process` but cover cases that we need to go to the parent table. In both cases you should take the DB related parts out and rely on injected dependencies to make this unit-testable.

Reply:
@gitcliff  you would refactor this and place all the db related code into the common submodule